File: WAT/WAT.Admin/Entities/Pax.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

namespace WAT.Admin.Entities
{
    [DataContract]
    public class Pax
    {
        public int PaxId { get; set; }

        [DataMember(Name ="apellido")]
        public string Apellido { get; set; }

        [DataMember(Name ="nombre")]
        public string Nombre { get; set; }

        [DataMember(Name ="email")]
        public string Email { get; set; }

        [DataMember(Name ="telefono")]
        public string Telefono { get; set; }

        [DataMember(Name ="fechaNacimiento")]
        public DateTime FechaNacimiento { get; set; }

        [DataMember(Name ="ciudad")]
        public int Ciudad { get; set; }

        [DataMember(Name ="universidad")]
        public int Universidad { get; set; }

        [DataMember(Name ="aceptaTerminos")]
        public bool AceptaTerminos { get; set; }
    }
}

File: WAT/WAT.Admin/Scripts/app/entities/Pax.js
function Pax() {
    this.paxId = null;
    this.apellido = '';
    this.nombre = '';
    this.email = '';
    this.telefono = '';
    this.fechaNacimiento = '';
    this.ciudad = new Item(0, 'Otro');
    this.universidad = new Item(0, 'Otro');
    this.aceptaTerminos = true;

    this.getDescripcion = function () {
        return this.apellido + ', ' + this.nombre;
    };
}

function Item(id, descripcion){
    this.id = id;
    this.descripcion = descripcion;
}